front-end/swift-buy/src/molecules/footer/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./footer.css";
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <div className="footer-sections">
+        <div className="footer-inner-sec">
+          <span className="footer-title">SwiftBuy</span>
```

Comment:
This SwiftBuy is the Title and it should be stored in utils.js file and from there we need to read